eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -149,6 +149,20 @@ jobs:
           ${{ insert }}: ${{ parameters.TestStepMavenInputs }}
         env: ${{ parameters.EnvVars }}
 
+      - task: Maven@3
+        displayName: 'Generate aggregate code coverage report'
+        condition: and(eq(variables['OsName'], 'Windows'), eq(variables['JavaTestVersion'], '1.11'), eq('${{ parameters.ServiceDirectory }}', 'communication'))
```

Comment:
We should avoid hardcoding checks specific to one service in our common templates. Instead we should have an opt-in or opt-out that can be set in the ci.yml for a given service.